src/items.js

```diff
@@ -1,26 +1,26 @@
 module.exports = [
 	{
-		label: 'Website',
+		label: '🌐 Website',
 		url: '{{ web }}'
 	},
 	{
-		label: 'Twitter',
+		label: '🐦 Twitter',
 		url: 'https://twitter.com/{{ twitter }}'
 	},
 	{
 		label: 'GitHub',
 		url: 'https://github.com/{{ github }}'
 	},
 	{
-		label: 'Blog',
+		label: '✍️ Blog',
 		url: '{{ blog }}'
 	},
 	{
-		label: 'Contact',
+		label: '📇 Contact',
 		url: 'https://{{ contact }}'
 	},
 	{
-		label: 'LinkedIn',
+		label: '💼 LinkedIn',
```

Comment:
Just what is this 😂 

Also please link a resource in readme where users can look for icons they wish to add for any of their items.

Reply:
This is work suitcase.
I think that would be https://emojipedia.org